### DISCUSSION TITLE
How Would I Use Obsidian To Write A Thesis

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#17] [Q&A] [answered]
> Hey Bryan thank you for this amazing explanation! Do you have any suggestion for using Zettelkasten/Evergreen notes for outlining and writing a thesis? Would you Split the chapter in folders first then add and link notes in there or just add a so-called “structure note”?

**caveat**: I've never written a thesis and I'm working on a bachelor's at the moment to hope for a master's after.

What I might do for a thesis is: 

1. Use Obsidian to get a bullet point outline of the thesis 
    + with the # headers for the chapters/sections of the thesis
    + and the bullets for the ideas points. 
    + all in one document for the outlining purposes
    + as the file grows and you flesh out the thesis
    + as this file grows in length, use the [note-refactor plugin](https://github.com/lynchjames/note-refactor-obsidian) to pull out the heading and content into new notes 
    + (organizing these notes would be made easier by prepending incrementing numbers like 1.0, 1.1, 1.1.3, etc to the beginning of the heading for the eventual refactoring). 

Keeping all of these thesis outline notes in a folder is likely the approach I would have just for a clear separation of concerns while I'm working on them 
OR 
you could have a transcluded note in your home note for instance I have `[[Top of Mind]]` for things I'm actively working on, and in that note have links to each of your thesis outline notes so no folders are needed:

```
![[1.0 First main idea]]
![[1.1 sub idea of first]]
![[1.1.3 granular details continued]]
etc..
```

That is what I would do for outlining using a Zettelkasten to inspire thesis content. 
For the actual writing of the thesis and organization of it, I'd likely use [RMarkdown](https://github.com/rstudio/rmarkdown) with the [{thesisdown}](https://github.com/ismayc/thesisdown) package 
OR
 just pure [LaTeX](https://www.latex-project.org/) and [Zotero](https://www.zotero.org/) for my citations .bib file  I have a whole playlist for [Rmarkdown](https://www.youtube.com/playlist?list=PL5fd4SsfvECyYYiSW3ZreC5GqIq8FBocx) and several [LaTeX](https://www.youtube.com/playlist?list=PL5fd4SsfvECx52dSPbI3XX5YD7fvuCaz-) videos too

**Comment by tallguyjenks** [comment, marked as answer]
Converted from issue

**Comment by mnott** [comment]
That's something that I may be able to contribute to. It is just a thought since I am a complete n00b when it comes to Obsidian, but as part of my writing, I've extensive experience with LaTeX, and hence was looking for a way to push the Zettelkasten idea to generate content for my PhD thesis.

I've been [Scrivener](https://www.literatureandlatte.com/) for a while now, and since I'm not only a student myself now, but also take care of some students, I'm mostly annoyed by it being limited to MacOS, paid software, etc. The file format is comparatively open - it's just a UI sitting on top of a bunch of rtf notes that are all held together by one big XML containing all the metadata.

What I've already done is to create a script on top of Scrivener - I called it [TeXDown](https://github.com/mnott/texdown) - that takes away most of the LaTeX command you'd otherwise use in your day to day work. I know there's markdown approaches already to it, but they were too limited for me, and I wanted more control - and integration with Scrivener actually, at that point in time.

Since I'm moving away from Scrivener and am gravitating towards Obsidian, I'm working on three things:

1. Learn much more about Obisidian so that I could make it work similar to what I have with Scrivener - that entails, of course, some tag approach as to what I want to actually include in any given LaTeX run (in Scrivener, you'd just check an "include in compile" checkbox on some node of your tree, and TeXDown will respect that; that's going to change as there's not necessarily a metadata file, but I could use a project file that just links in some starter nodes in the tree).
2. Then I'll expand TeXDown to do all of that scripting / preparing the LaTeX source that I want
3. Finally, wrap everything up in a docker container, to make it easier for people using it - including LaTeX itself.

Let me know what you think. It's work in progress, but maybe you already see that there's basically stuff others have done already that can serve as a shortcut.

M

**Reply by tallguyjenks** [reply]
I'm not quite sure what your aim is exactly? 

What problem are you trying to solve? Or what is causing friction that you're trying to smooth over?
